### PR TITLE
Fix websocket patch

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

File: zone.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ Zone.patchSetClearFn = function (obj, fnNames) {
   });
 };
 
-Zone.nextId = 1;
+Zone.nextId = 0;
```
Comment (btford) [inline zone.js]: root zone should still be 1

**Comment by caitp** [comment]
> TypeError: Attempting to configurable attribute of unconfigurable property. in /home/travis/build/angular/zone.js/zone.js (line 604)

The original "canPatchPropertyDescriptor" fix didn't resolve that? :(

**Comment by vicb** [comment]
@IgorMinar great changes !

wrt id numbering, may be we can cut a bug fix release w/o it (0.4.next) and change it for the next version (0.5) which hopefully will be the "modularized" version ?

**Comment by btford** [comment]
@vicb – the API is (in theory) private – do you know of anywhere it's used?

**Comment by vicb** [comment]
I'm not aware of any usage and I would love to this change. But that is not something we need, it's only cosmetic. So I think it could probably wait a little bit.

**Comment by btford** [comment]
Hm, okay. @vicb – can you merge the other commits in this PR and create a new PR with just the root zone id fix?

**Comment by vicb** [comment]
Today is labor day here, I'm not allowed to work ;)
No pb if it can wait until Monday.

**Comment by btford** [comment]
Commenting on GitHub is working!

Go enjoy your holiday :P

**Comment by IgorMinar** [comment]
The current patch is not correct. I'm fixing it up.

**Comment by IgorMinar** [comment]
I created #87 for the zone.id change per @vicb's request.

**Comment by IgorMinar** [comment]
@caitp 

> TypeError: Attempting to configurable attribute of unconfigurable property. in /home/travis/build/angular/zone.js/zone.js (line 604)

Is due to #88 and affects only Safari 7.0 but not Safari 7.1.

This will be hard to test because I don't have 7.0 and web sockets don't work via SL (BrowserStack might work though).

**Comment by btford** [comment]
LGTM, merging.

**Comment by btford** [comment]
landed as f01263a, 1799a20, e5d8c22, edb17d2, d725f46

**Pull request closed without merging**